src/components/Layout/Footer/Footer.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { action } from "@storybook/addon-actions";
+import { text, array } from "@storybook/addon-knobs";
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+
+import Footer from "./Footer";
+
+<Meta title="Library/Layout/Footer" component={Footer} />
+
+# MeetingCard
+
+### Displays a card
```

Comment:
Could you remove or change it to `# Footer`?

Reply:
These two lines actually show up in the docs panel, so need to change accordingly.

![image](https://user-images.githubusercontent.com/37560480/112571625-b0c8c500-8da5-11eb-90f2-b609523334fd.png)

Reply:
@tohuynh Just made these changes. My [gh-pages](https://andrewlubrino.github.io/cdp-frontend/?path=/docs/library-layout-footer--footer) is also updated. 

I only added the documentation comments to `footerLinksSections`. I tried adding documentation comments to other objects that are used indirectly in the `Footer` props (so `links`, `footerLinksSectionName`, and then the members that comprise the `Link` interface, `label` and `url`), but it wasn't showing up in the storybook docs, so I took them out for now. 

Let me know if this works.

Reply:
Looks good. I think the comments only show for component's direct props.